<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">



<title>Comprehensive genome-wide analysis of transposable elements for its association with disease resistance genes in rice (Oryza sativa L)</title>

<script src="EMBO21_ravi_files/header-attrs-2.10/header-attrs.js"></script>






<!--
Font-awesome icons ie github or twitter
-->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/brands.css" integrity="sha384-n9+6/aSqa9lBidZMRCQHTHKJscPq6NW4pCQBiMmHdUCvPN8ZOg2zJJTkC7WIezWv" crossorigin="anonymous">

<!--
Google fonts api stuff
-->
<link href='https://fonts.googleapis.com/css?family=Special Elite' rel='stylesheet'>
<link href='https://fonts.googleapis.com/css?family=Rasa' rel='stylesheet'>

<!--
Here are the required style attributes for css to make this poster work :)
-->
<style>
@page {
size: 65in 195in;
margin: 0;
padding: 0;
}
body {
margin: 0px;
padding: 0px;
width: 65in;
height: 195in;
text-align: justify;
font-size: 45px;
}
/* RMarkdown Class Styles */
/* center align leaflet map,
from https://stackoverflow.com/questions/52112119/center-leaflet-in-a-rmarkdown-document */
.html-widget {
margin: auto;
position: sticky;
margin-top: 2cm;
margin-bottom: 2cm;
}
.leaflet.html-widget.html-widget-static-bound.leaflet-container.leaflet-touch.leaflet-fade-anim.leaflet-grab.leaflet-touch-drag.leaflet-touch-zoom {
position: sticky;
width: 100%;
}
pre.sourceCode.r {
background-color: #dddddd40;
border-radius: 4mm;
padding: 4mm;
width: 75%;
margin: auto;
margin-top: 1em;
margin-bottom: 1em;
/* align-items: center; */
padding-left: 2cm;
}
code.sourceCode.r{
background-color: transparent;
font-size: 20pt;
border-radius: 2mm;
}
code {
font-size: 25pt;
font-family: monospace;
background-color: #FF677124;
color: #DA2536;
padding: 1.2mm;
border-radius: 2mm;
}
caption {
margin-bottom: 10px;
font-size: 20pt;
font-style: italic;
}
tbody tr:nth-child(odd) {
    background-color: #DA253620;
}
.table>thead>tr>th, .table>tbody>tr>th, .table>tfoot>tr>th, .table>thead>tr>td, .table>tbody>tr>td, .table>tfoot>tr>td{
  border-spacing: 0;
  border-style: none;
  padding-top: 15px;
  padding-bottom: 15px;
  padding-right: 20px;
  padding-left: 20px;
  line-height: 1.42857143;
}
.caption {
font-size: 20pt;
font-style: italic;
}
.references {
font-size: 23px;
line-height: 90%;
}
/* Create three unequal columns that floats next to each other */
.column {
float: left;
padding: 0px;
}
.outer {
width: 65in;
height: calc(195in * 0.6325);
-webkit-column-count: 3; /* Chrome, Safari, Opera */
-moz-column-count: 3; /* Firefox */
column-count: 3;
-webkit-column-fill: auto;
-moz-column-fill: auto;
column-fill: auto;
column-gap: 0;
padding-left: 0cm;
padding-right: 0cm;
/* -webkit-column-rule-width: 50%;
-moz-column-rule-width: 50%;
column-rule-width: 50%; */
-webkit-column-rule-style: none;
-moz-column-rule-style: none;
column-rule-style: none;
-webkit-column-rule-color: black;
-moz-column-rule-color: black;
column-rule-color: black;
background-color: #ffffff;
font-family: Rasa;
margin-top: calc(195in * 0.25);
padding-top: 1em;
padding-bottom: 1em;
}
span.citation {
  color: #FF6771;
  font-weight: bold;
}
a {
text-decoration: none;
color: #FF6771;
}
#title {
font-size: 70pt;
text-align: left;
padding-right: 5%;
padding-left: 5%;
margin: 0;
line-height: 98%;
border-bottom: 0;
font-weight: normal;
}
#author {
padding-right: 5%;
padding-left: 5%;
color: #DA2536;
margin: 0;
line-height: 85%;
font-size: 1.17em;
}
#affiliation {
padding-right: 5%;
padding-left: 5%;
padding-top: 1em;
color: #00000080;
font-style: italic;
font-size: 25px;
margin: 0;
}
sup {
color: #5d19c4;
}
.affiliation sup {
font-size: 20px;
}
table {
  padding-left: 5%;
  padding-right:; 5%;
}
.author {
text-align: left;
}
.author sup {
font-size: 30px;
}
.author_extra {
padding-right: 5%;
padding-left: 5%;
color: #FF6771;
margin: 0;
line-height: 85%;
font-size: 35px;
text-align: left;
}
.outer h1, h2, h3, h4, h5, h6 {
text-align: center;
margin-top: 0.5in;
margin-bottom: 0.5in;
font-weight: normal;
}
.outer h2 {
text-align: center;
}
.outer p, .level2 {
padding-right: 5%;
padding-left: 5%;
color: #000000;
}
.outer ol {
padding-left: 8%;
padding-right: 8%;
text-align: left;
}
.middle {
width: 65in;
height: calc(195in * 0.25);
position: absolute;
background-color: #DA2536;
color: #FFFFFF;
font-family: Special Elite;
background-image: linear-gradient(#DA2536 50%, #FF6771);
}
.middle strong {
color: #FFFFFF;
}
.middle strong > sup {
color: #FFFFFF;
}
.middle sup {
color: #FFFFFF;
}
#middle-img-left {
width: 20%;
left: 0.5in;
bottom: 0.2in;
position: absolute;
opacity: 1
}
#middle-img-center {
width: 10%;
left: calc(65in * 0.45);
bottom: 0.5in;
position: absolute;
opacity: 0.5
}
#middle-img-right {
width: 10%;
right: 0.5in;
bottom: 0.2in;
position: absolute;
opacity: 1
}
.middle p {
padding-left: 7%;
padding-right: 7%;
font-size: 150px;
text-align: left;
}
.fab {
color: #00000030;
font-size: 25px;
}
.twitter, i {
color: #00000030;
font-size: 35px;
text-decoration: none;
}
a.email {
text-decoration: none;
color: #00000030;
font-size: 35px;
}
.envelope {
color: #00000030;
font-size: 5px;
text-decoration: none;
}
.poster_wrap {
width: 65in;
height: 195in;
padding: 0cm;
}
.emphasis_bottom {
width: 65in;
height: calc(195in * 0.1);
margin-top: calc(195in * 0.9);
position: absolute;
background-color: #DA2536;
background-image: linear-gradient(#FF6771 10%, #DA2536);
}

span > #tab:mytable {
  font-weight: bold;
}
</style>
</head>
<body>


<div class="poster_wrap">

<div class="column outer">
<h1 id="title"><strong>Comprehensive genome-wide analysis of transposable elements for its association with disease resistance genes in rice (<em>Oryza sativa</em> L)</strong></h1><br>
<h3 id="author" class="author">

Ravindra Raut<sup> 1</sup><br>
<a class="twitter" href="https://mobile.twitter.com/Ravindra__Raut"><i class="fab fa-twitter"></i>&nbsp;&nbsp;@Ravindra__Raut</a><br>

<a class='envelope'><i class="fas fa-envelope"></i></a> <a href="mailto:raviraut206@gmail.com" class="email">raviraut206@gmail.com</a> <br>
  </h3>

<h5 id="author_extra", class="author_extra">
 Subhankar Roy Barman<sup>1</sup>
 Malali Gowda<sup>2</sup>
</h5>

<p id="affiliation" class="affiliation">
<sup>1</sup> Department of Biotechnology, National Institute of Technology Durgapur, India<br> <sup>2</sup> Center for Functional Genomics and Bioinformatics, The University of Health Sciences and Technology Bengaluru, India
</p>

<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<ul>
<li>Transposable elements (TE) or transposons are mobile DNA elements which exist in almost all eukaryotic genomes [1].</li>
<li>Firstly these elements are identified and described by geneticist Barbara McClintock.</li>
<li>TEs are categorized based on the intermediate substrate producing insertions (DNA/RNA) [2]</li>
<li><code>Class I (retrotransposons)</code>: SINEs [ALUs, MIRs, Penelope], LINEs [L1, L2, L3/CR1, R1/LOA/Jockey], LTRs [ERVL, Ty1/Copia, Gypsy/DIRS1]</li>
<li><code>Class II (DNA transposons)</code>: hAT, En-Spm, MuDR, Tourist/Harbinger, Mirage, P-element, Transib.</li>
</ul>
</div>
<div id="objectives" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Objectives</h1>
<ul>
<li>To study the Transposable Elements (TEs) in different rice genomes: (<em>Oryza sativa</em> Japonica, <em>Oryza sativa</em> indica and wild rice species).</li>
<li>To study association of TEs and Plant Resistance (R)-genes.</li>
</ul>
</div>
<div id="methods" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Methods</h1>
<p>The de novo repeat prediction performed using <code>RepeatModeler 1.0.11</code>. The program runs with RM-BLAST (NCBI) database as input for repeat modelling. At the core of RepeatModeler, there are programs called RECON, RepeatScout, Tandem Repeat Finder. RepeatModeler follows the nomenclature code of Dfam to classify TEs. The library file generated by RepeatModeler was used to estimate the coverage of different classes of repeats by the <code>RepeatMasker 4.1.0</code>.</p>
</div>
<div id="results" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Results</h1>
<p>The rice genome assemblies are investigated to detect and identify the presence of transposable elements include previously published genomes; we selected thirty rice assemblies based on genome coverage that includes Oryza sativa Japonica Group, Indica Group, African wild rice, Australian wild rice, South American rice wild species and African outgroup species <em>Leersia perrieri</em> (Launert or Cutgrass) Figure <a href="#fig:Figure-1">4.1</a>.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Figure-1"></span>
<img src="Figures/Figure-1.png" alt="The percentage of transposable elements in oryza genome and classification of assemblies based on the genomic groups." width="100%" />
<p class="caption">
Figure 4.1: The percentage of transposable elements in oryza genome and classification of assemblies based on the genomic groups.
</p>
</div>
<div id="transposable-elements-classes-in-rice-genome-assemblies" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Transposable Elements Classes in rice genome assemblies</h2>
<p>This study revealed unique pattern of TE Classes in rice genome assemblies (Figure <a href="#fig:Figure-2">4.2</a>).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Figure-2"></span>
<img src="Figures/Figure-2.png" alt="Analysis of rice genome assemblies for (A) Retroelements, (B) DNA elements, (C) Total interspersed repeats, (D) SINEs, (E) LINEs, and (F) LTR elements and (G) TE superfamily." width="90%" />
<p class="caption">
Figure 4.2: Analysis of rice genome assemblies for (A) Retroelements, (B) DNA elements, (C) Total interspersed repeats, (D) SINEs, (E) LINEs, and (F) LTR elements and (G) TE superfamily.
</p>
</div>
</div>
<div id="association-of-tes-and-plant-resistance-r-genes" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> Association of TEs and Plant Resistance (R)-genes</h2>
<p>Plant Resistance (R)-genes in rice encode products that recognize pathogens and triggers disease resistance, but its’ expression can be influences by TEs.For eg. RPM1-Disease Resistance Gene have TE footprit sequences in its exonic regions, protein product and in protein domain (Figure <a href="#fig:Figure-3">4.3</a>), (Figure <a href="#fig:Figure-4">4.4</a>), (Figure <a href="#fig:Figure-5">4.5</a>).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Figure-3"></span>
<img src="Figures/Figure-3.png" alt="TE footprints sequences in RPM1-Disease Resistance Gene" width="95%" />
<p class="caption">
Figure 4.3: TE footprints sequences in RPM1-Disease Resistance Gene
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Figure-4"></span>
<img src="Figures/Figure-4.png" alt="TE footprints in Rx-N domain of RPM1-Disease Resistance Protein" width="95%" />
<p class="caption">
Figure 4.4: TE footprints in Rx-N domain of RPM1-Disease Resistance Protein
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Figure-5"></span>
<img src="Figures/Figure-5.png" alt="The 3D structure of  RPM1-Disease Resistance Proteins (A. Os04g46300.1 and B. Os08g28470.1) showing TE footprints inside Rx-N domain" width="95%" />
<p class="caption">
Figure 4.5: The 3D structure of RPM1-Disease Resistance Proteins (A. Os04g46300.1 and B. Os08g28470.1) showing TE footprints inside Rx-N domain
</p>
</div>
</div>
</div>
<div id="conclusions" class="section level1" number="5">
<h1><span class="header-section-number">5</span> Conclusions</h1>
<ul>
<li>This study revealed the Repeats, constitute <code>20.47-48.55%</code> of the rice genome assemblies, with average <code>39.01%</code>.</li>
<li>The LTR class <code>Ty1/Copia</code> and <code>Gypsy/DIRS1</code> account for a significant portion of the genome.</li>
<li>Some TEs have association with Rice disease resistance (R) genes.</li>
</ul>
</div>
<div id="references" class="section level1" number="6">
<h1><span class="header-section-number">6</span> References</h1>
<ol style="list-style-type: decimal">
<li>Xiong (1990). <a href="https://doi.org/10.1002/j.1460-2075.1990.tb07536.x" class="uri">https://doi.org/10.1002/j.1460-2075.1990.tb07536.x</a></li>
<li>Wicker (2007). <a href="https://doi.org/10.1038/nrg2165" class="uri">https://doi.org/10.1038/nrg2165</a></li>
</ol>
</div>

</div>
<div class="column middle">
<p>Comprehensive genome-wide analysis of transposable elements in 30 diferent cultivars and wild rice species; reveals LTR elements are major repeat constitute</p>
</div>
<div class="emphasis_bottom">
<img id="middle-img-left" src=Figures/NIT_Durgapur_logo.png>
<img id="middle-img-center" src=>
<img id="middle-img-right" src="Figures/TDULogo21.jpg">
</div>
</div>



</body>
</html>
