---
# PLEASE SEE THE README for in depth description github.com/brentthorne/posterdown
poster_height: "48in"
poster_width: "36in"
font_family: 'Rasa'
#ESSENTIALS
title: '**Comprehensive genome-wide analysis of transposable elements for its association with disease resistance genes in rice (*Oryza sativa* L)**'
author:
  - name: Ravindra Raut
    affil: 1
 
    main: true
    orcid: 0000-0002-2615-7757
    twitter: Ravindra__Raut
    email: raviraut206@gmail.com  
      
  - name: Subhankar Roy Barman
    affil: 1
  - name: Malali Gowda
    affil: 2
  

affiliation:
  - num: 1
    address: Department of Biotechnology, National Institute of Technology Durgapur, India
  - num: 2
    address: Center for Functional Genomics and Bioinformatics, The University of Health Sciences and Technology Bengaluru, India

#STYLE & FORMATTING
title_textsize: "70pt"
author_textsize: "1.17em"
authorextra_textsize: "35px"
affiliation_textsize: "25px"
affiliation_textcol: '#00000080'
caption_fontsize: "20pt"
#Middle of the poster
middle_fontfamily: "Special Elite"
middle_textcol: "#FFFFFF"
middle:textalpha: 90
middle_fontsize: "150px"
main_findings:
  - " Comprehensive genome-wide analysis of transposable elements in 30 diferent cultivars and wild rice species; reveals LTR elements are major repeat constitute"
logoleft_name: "Figures/NIT_Durgapur_logo.png"
logocenter_name: ""
logoright_name: "Figures/TDULogo21.jpg"
#---POSTER BODY OPTIONS---#
primary_colour: '#DA2536'
secondary_colour: '#FF6771'
accent_colour: "#5d19c4"
body_bgcol: "#ffffff"
body_textsize: "45px"
body_textcol: "#000000"
reference_textsize: "23px"
#--Standard Options--#
output: 
  posterdown::posterdown_betterport:
    self_contained: false
    pandoc_args: --mathjax
    highlight: espresso
    number_sections: true
    template: template.html
bibliography: "/Users/ravindraraut/Downloads/RStudio_work/recount-brain-bog19-master/MyBib.bib"
---
```{r setup, include=FALSE}
knitr::opts_chunk$set(
  results = 'asis',
  echo = FALSE,
  comment = FALSE,
  warning = FALSE,
  message = FALSE,
  fig.align = 'center'
)
```

# Introduction

* Transposable elements (TE) or transposons are mobile DNA elements which exist in almost all eukaryotic genomes [1].
* Firstly these elements are identified and described by geneticist Barbara McClintock.
* TEs are categorized based on the intermediate substrate producing insertions (DNA/RNA) [2]
* `Class I (retrotransposons)`: SINEs [ALUs, MIRs, Penelope], LINEs [L1, L2, L3/CR1, R1/LOA/Jockey], LTRs [ERVL, Ty1/Copia, Gypsy/DIRS1]
* `Class II (DNA transposons)`: hAT, En-Spm, MuDR, Tourist/Harbinger, Mirage, P-element, Transib.

# Objectives

* To study the Transposable Elements (TEs) in different rice genomes: (*Oryza sativa* Japonica, *Oryza sativa* indica and wild rice species).
* To study association of TEs and Plant Resistance (R)-genes.

# Methods

The de novo repeat prediction performed using `RepeatModeler 1.0.11`. The program runs with RM-BLAST (NCBI) database as input for repeat modelling. At the core of RepeatModeler, there are programs called RECON, RepeatScout, Tandem Repeat Finder. RepeatModeler follows the nomenclature code of Dfam to classify TEs. The library file generated by RepeatModeler was used to estimate the coverage of different classes of repeats by the `RepeatMasker 4.1.0`.

# Results
The rice genome assemblies are investigated to detect and identify the presence of transposable elements include previously published genomes; we selected thirty rice assemblies based on genome coverage that includes Oryza sativa Japonica Group, Indica Group, African wild rice, Australian wild rice, South American rice wild species and African outgroup species *Leersia perrieri* (Launert or Cutgrass) Figure \@ref(fig:Figure-1).


```{r Figure-1, out.width='100%', fig.cap='The percentage of transposable elements in oryza genome and classification of assemblies based on the genomic groups.'}
knitr::include_graphics('/Users/ravindraraut/Downloads/RStudio_work/recount-brain-bog19-master/Figures/Figure-1.png')
```

## Transposable Elements Classes in rice genome assemblies 

This study revealed unique pattern of TE Classes in rice genome assemblies (Figure  \@ref(fig:Figure-2)).

```{r Figure-2, out.width='90%', fig.cap='Analysis of rice genome assemblies for (A) Retroelements, (B) DNA elements, (C) Total interspersed repeats, (D) SINEs, (E) LINEs, and (F) LTR elements and (G) TE superfamily.'}
knitr::include_graphics('/Users/ravindraraut/Downloads/RStudio_work/recount-brain-bog19-master/Figures/Figure-2.png')
```

## Association of TEs and Plant Resistance (R)-genes
Plant Resistance (R)-genes in rice encode products that recognize pathogens and triggers disease resistance, but itsâ€™ expression can be influences by TEs.For eg. RPM1-Disease Resistance Gene have TE footprit sequences in its exonic regions, protein product and in protein domain (Figure  \@ref(fig:Figure-3)), (Figure \@ref(fig:Figure-4)), (Figure \@ref(fig:Figure-5)).

```{r Figure-3, out.width='95%', fig.cap='TE footprints sequences in RPM1-Disease Resistance Gene'}
knitr::include_graphics('/Users/ravindraraut/Downloads/RStudio_work/recount-brain-bog19-master/Figures/Figure-3.png')
```

```{r Figure-4, out.width='95%', fig.cap='TE footprints in Rx-N domain of RPM1-Disease Resistance Protein'}
knitr::include_graphics('/Users/ravindraraut/Downloads/RStudio_work/recount-brain-bog19-master/Figures/Figure-4.png')
```

```{r Figure-5, out.width='95%', fig.cap='The 3D structure of  RPM1-Disease Resistance Proteins (A. Os04g46300.1 and B. Os08g28470.1) showing TE footprints inside Rx-N domain'}
knitr::include_graphics('/Users/ravindraraut/Downloads/RStudio_work/recount-brain-bog19-master/Figures/Figure-5.png')
```

# Conclusions

* This study revealed the Repeats, constitute `20.47-48.55%`  of the rice genome assemblies, with average `39.01%`.
* The LTR class `Ty1/Copia`  and `Gypsy/DIRS1` account for  a significant portion of the genome.
* Some TEs have association with Rice disease resistance (R) genes.

# References
1. Xiong (1990). https://doi.org/10.1002/j.1460-2075.1990.tb07536.x
2. Wicker (2007). https://doi.org/10.1038/nrg2165

